backend/app/api/endpoints/transcription.py

```
@@ -16,7 +15,7 @@ async def transcribe_audio(
     audio: UploadFile = File(...),
     language: str = "en-US",
     enable_punctuation_commands: bool = False,
-    current_user: User = Depends(get_current_user_from_session)
+    current_user: Dict = Depends(get_current_user_from_session)
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)